### ISSUE TITLE
Validation and testing results are not good

### ISSUE THREAD

**Issue by chienyiwang** [gafniguy/4D-Facial-Avatars#18]
Hi,
Thanks for the great work and kind release of the implementation. I followed the modification in [Issue#16](https://github.com/gafniguy/4D-Facial-Avatars/issues/16) and could train the model with your default .yml file. However, the validation and testing results are not good visually. Most of them are blurry and some of them even only contains background. The training losses are decreasing, but the coarse loss of validation set is not. Could you please provide some suggestions about how to tune the parameters or fix the model training? Thanks a lot!

The validation results and loss curves from tensorboard snapshots are attached below: 
![test_result](https://user-images.githubusercontent.com/15913177/152935568-18c5affc-3f08-4c39-8bed-327ba760dc2f.PNG)
![train_loss](https://user-images.githubusercontent.com/15913177/152935585-171b7dda-3066-40e4-b5d0-406f7b4c67e2.PNG)

**Comment by chienyiwang** [comment]
I accidentally set "no_ndc" to "False" in the previous experiments... After setting back "no_ndc" to "True", the results are getting much better now. 
![result](https://user-images.githubusercontent.com/15913177/153327097-c1d137fd-c60c-4910-9144-6e7d17a1d039.PNG)
![loss](https://user-images.githubusercontent.com/15913177/153327101-2016b208-ca20-437b-bdec-af5b72909f6b.png)

**Comment by chienyiwang** [comment]
If possible, could you elaborate more on the choice of "no_ndc"? Is it heavily dependent on the input format or just empirical? Thanks a lot!

**Comment by Alpe6825** [comment]
> I accidentally set "no_ndc" to "False" in the previous experiments... After setting back "no_ndc" to "True", the results are getting much better now. ![result](https://user-images.githubusercontent.com/15913177/153327097-c1d137fd-c60c-4910-9144-6e7d17a1d039.PNG) ![loss](https://user-images.githubusercontent.com/15913177/153327101-2016b208-ca20-437b-bdec-af5b72909f6b.png)

I get similar results as in the fist picture in that issue even if no_ndc is true.

**Comment by jianweif** [comment]
Same here. Trained with person_1's data and config. Training psnr gradually reaches 32 while validation psnr plateaus at 20. Would really appreciate if author can help here!

**Comment by Volleria** [comment]
> Hi, Thanks for the great work and kind release of the implementation. I followed the modification in [Issue#16](https://github.com/gafniguy/4D-Facial-Avatars/issues/16) and could train the model with your default .yml file. However, the validation and testing results are not good visually (using ckpt at epoch 500k/600k/...). Most of them are blurry and some of them even only contains background. The training losses are decreasing, but the coarse loss of validation set is not. Could you please provide some suggestions about how to tune the parameters or fix the model training? Thanks a lot!
> 
> The validation results and loss curves from tensorboard snapshots are attached below: ![test_result](https://user-images.githubusercontent.com/15913177/152935568-18c5affc-3f08-4c39-8bed-327ba760dc2f.PNG) ![train_loss](https://user-images.githubusercontent.com/15913177/152935585-171b7dda-3066-40e4-b5d0-406f7b4c67e2.PNG)

Hi！How did you get these charts？

**Comment by Volleria** [comment]
> Hi, Thanks for the great work and kind release of the implementation. I followed the modification in [Issue#16](https://github.com/gafniguy/4D-Facial-Avatars/issues/16) and could train the model with your default .yml file. However, the validation and testing results are not good visually (using ckpt at epoch 500k/600k/...). Most of them are blurry and some of them even only contains background. The training losses are decreasing, but the coarse loss of validation set is not. Could you please provide some suggestions about how to tune the parameters or fix the model training? Thanks a lot!
> 
> The validation results and loss curves from tensorboard snapshots are attached below: ![test_result](https://user-images.githubusercontent.com/15913177/152935568-18c5affc-3f08-4c39-8bed-327ba760dc2f.PNG) ![train_loss](https://user-images.githubusercontent.com/15913177/152935585-171b7dda-3066-40e4-b5d0-406f7b4c67e2.PNG)

Hi！How did you get these charts？

**Comment by wuzuyin** [comment]
> 同样在这里。使用 person_1 的数据和配置进行训练。训练 psnr 逐渐达到 32，而验证 psnr 稳定在 20。如果作者能在这里提供帮助，我将不胜感激！

你好，我也遇到了同样的问题，请问你解决了吗

**Comment by wuzuyin** [comment]
> Same here. Trained with person_1's data and config. Training psnr gradually reaches 32 while validation psnr plateaus at 20. Would really appreciate if author can help here!

Hello, I have also encountered the same problem. Have you resolved it
